cypress/page-objects/lib/button.ts

```diff
@@ -10,4 +14,8 @@ export class Button extends UIElement {
   getButtonByText(text: string) {
     return this.getElement().contains(text)
   }
+
+  getFilter(text: string) {
```

Comment:
If abstract from the current app then it's hard to understand what `button.getFilter` means. Usually there is no filter for buttons. My point here is: framework API should be understandable.